docs/data/whats-new/WhatsNewLayout.tsx

```diff
@@ -20,6 +20,39 @@ type Blog = {
 };
 
 const blogs: Blog[] = [
+  {
+    title: 'MUI X v6.18.0',
+    description:
+      'New stable components, polished features, better performance and more.',
+    announcementDate: 'Monday, Nov 13, 2023',
+    url: 'https://mui.com/blog/mui-x-end-v6-features/',
+    highlightList: [
+      {
+        title: 'Charts - stable version',
+        url: 'https://mui.com/blog/mui-x-end-v6-features/#charts',
+      },
+      {
+        title: 'Tree view - stable version',
+        url: 'https://mui.com/blog/mui-x-end-v6-features/#tree-view',
+      },
+      {
+        title: 'Date and time clearable fields',
+        url: 'https://mui.com/blog/mui-x-end-v6-features/#clearable-field',
+      },
+      {
+        title: 'Customization Playgrounds for date and time pickers',
```

Comment:
```suggestion
        title: 'Customization playgrounds for Date and Time Pickers',
```

But it's only available for `DatePicker` for now right?